build.sbt

```diff
@@ -1,7 +1,36 @@
-name := "sbt-dcos"
+import com.mesosphere.sbt.BuildPlugin.publishSettings
```

Comment:
Similarly, I've made the package for the plugin `com.mesosphere.sbt`. Would it be more appropriate as `io.dcos.sbt`? Counterpoint: Cosmos uses `com.mesosphere.cosmos`.